lib/aggregate.js

```diff
@@ -19,6 +19,7 @@ module.exports = function(sourcePath) {
             });
 
             return {
+                event: "coverage",
```

Comment:
Since the aggregate is "just" a datastructure, with [a specification](https://github.com/zwilias/elm-coverage/blob/master/docs/elm-coverage.json) and whatnot, I'd rather not consider it an event as such. Perhaps it makes more sense to add the `event` key in the `runner` before displaying it?

Just adding it to the `data` [before writing it to the output stream](https://github.com/zwilias/elm-coverage/pull/7/files#diff-b3b53682a18f203ac8d29b0e277cad26R218) would most likely do the trick.